bindings/tflite/README.md

```diff
@@ -68,7 +68,7 @@ ability to directly execute pages from the memory.
 ## Support
 
 |   | Glossary
-| - | --------
+| --- | --------
```

Comment:
I'm actually pretty unconvinced by this change and commented as much on the internal change. It makes the table less readable in source imo. We could apply formatting to get everything to align, which would probably satisfy the internal parser and look nicer in source

Reply:
https://docs.github.com/en/github/writing-on-github/organizing-information-with-tables

> Cells can vary in width and do not need to be perfectly aligned within columns. There must be at least three hyphens in each column of the header row.

Reply:
Ah interesting that it was working without that then. I sent https://github.com/google/iree/pull/5773 to clean this up